bundled/boost-1.62.0/include/boost/container/detail/copy_move_algo.hpp

```diff
@@ -34,7 +34,12 @@
 // other
 #include <boost/core/no_exceptions_support.hpp>
 // std
-#include <cstring> //for emmove/memcpy
+#include <cstring> //for memmove/memcpy
+
+#if defined(BOOST_GCC) && (BOOST_GCC >= 80000)
+#pragma GCC diagnostic push
+#pragma GCC diagnostic ignored "-Wclass-memaccess"
+#endif
```

Comment:
Isn't it necessary to have a `pragma pop` somewhere?

Reply:
Yes, fixed.